package.json

```diff
@@ -477,6 +477,35 @@
                 "url": "http://json.schemastore.org/proxies"
             }
         ],
+        "problemPatterns": [
```

Comment:
What happens with current projects that have a conflicting problem matcher defined?

Reply:
Here's what an old project looks like:
```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "Run Functions Host",
      "type": "shell",
      "command": "func host start",
      "isBackground": true,
      "problemMatcher": [
        {
          "owner": "azureFunctions",
          "pattern": [
            {
              "regexp": "\\b\\B",
              "file": 1,
              "location": 2,
              "message": 3
            }
          ],
          "background": {
            "activeOnStart": true,
            "beginsPattern": "^.*Stopping host.*",
            "endsPattern": "^.*Job host started.*"
          }
        }
      ]
    }
  ]
}
```

Here's what a new project looks like:
```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "Run Functions Host",
      "type": "shell",
      "command": "func host start",
      "isBackground": true,
      "problemMatcher": "$func-watch"
    }
  ]
}
```
Old projects will continue to use that "hard-coded" problem matcher as if nothing had changed. Only new projects will benefit from "func-watch"